config.json.example

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "ServerName",
+    "authServer": "https://api.kocity.xyz",
+    "publicAddr": "127.0.0.1:23600",
+    "maxPlayers": 50,
+    "external": {
+        "port": 23600
+    },
+    "internal": {
+        "host": "127.0.0.1",
+        "port": 23500
+    },
+    "redis": {
+        "host": "127.0.0.1",
+        "port": 6380,
+        "password": ""
+    },
+    "postgres": "postgres://viper:viper@127.0.0.1:5434/viper",
+
+    "statsWatcher": {
+        "enabled": false,
+        "gameDbUrl": "postgres://viper:viper@127.0.0.1:5434/viper",
```

Comment:
Why would I need to reconfigure this URL? Its already configured as `postgres` / `DATABASE_URL`. Why the duplication here?

If I understand correctly this will always point to the same database regardless.